hypertrace-gradle-java-convention-plugin/src/main/java/org/hypertrace/gradle/java/convention/JavaConventionPlugin.java

```diff
@@ -31,6 +39,7 @@ private void createConvention(Project target) {
   }
 
   private void configureCompatibility(Project target) {
+    // TODO: resolve about usage of `.get()`
     JavaVersion javaVersion = javaConventionExtension(target).compatibilityVersion.get();
     javaPluginExtension(target).setSourceCompatibility(javaVersion);
     javaPluginExtension(target).setTargetCompatibility(javaVersion);
```

Comment:
I can't directly comment on it because it hasn't changed, but on lines 51-53, what are we doing
```
    javaToolchainService(target).compilerFor(spec);
    javaToolchainService(target).launcherFor(spec);
    javaToolchainService(target).javadocToolFor(spec)
```
Those are getters, not setters I thought?